gax-grpc/src/test/java/com/google/api/gax/grpc/CancellationHelpers.java

```diff
@@ -47,7 +47,9 @@ public static void cancelInThreadAfterLatchCountDown(
               public void run() {
                 try {
                   latch.await();
-                  while (!resultFuture.cancel(true)) {
+                  System.out.println("Cancelling");
+                  while (!resultFuture.cancel(true) && !resultFuture.isDone()) {
+                    System.out.println("Cancelling again");
```

Comment:
If we are going to keep these print statements, maybe make this more informative (e.g. why is it cancelling again?)